ibis/common/typing.py

```diff
@@ -125,8 +125,8 @@ def get_bound_typevars(obj: Any) -> dict[str, Any]:
 
     Returns
     -------
-    dict[str, Any]
-        Mapping of type variable name to type.
+    dict[TypeVar, tuple[str, type]]
```

Comment:
We should remove the type annotation from the docstring.

Reply:
You mean remove all type annotations from the docstrings or specifically this one?